adrs/0153-checkout-v2.md

```diff
@@ -40,9 +40,12 @@ We want to take this opportunity to make behavioral changes, from v1. This docum
     default: ${{ github.token }}
   ssh-key:
```

Comment:
I wonder whether a separate input `prefer-token` would be better than this implicit behavior.